### ISSUE TITLE
Remove open external app pop up for phone numbers 

### ISSUE THREAD

**Issue by anthonypkeane** [brave/brave-ios#2457]
### Description: 
Follow up to:https://github.com/brave/brave-ios/issues/551

Please remove the open external app popup option for phone numbers as it ads 30% more taps

Also, it is expected behavior on iOS that tapping on a phone number would dial it

**Remove this pop up for phone numbers**
![Image from iOS (1).png](https://images.zenhubusercontent.com/5b919b7b7859ca34f6abcb40/703df9f5-e5c7-45f7-aa20-46b61ecc1dcd)

@jumde can you please review?

**Comment by jumde** [comment]
@anthonypkeane - I don't think this is a good idea. `tel+` links are often used as a way to phish users to call premium numbers. This prompt notifies them that a link in webpage is trying to invoke a call. More details here: https://nakedsecurity.sophos.com/2014/09/03/apple-developer-guidelines-lead-to-rogue-phone-call-risks-in-ios/

**Comment by anthonypkeane** [comment]
How does Safari deal with this? Thanks

**Comment by jumde** [comment]
Safari also has a popup that requires user confirmation to dial a number.

**Comment by anthonypkeane** [comment]
Here is an example: 

Safari
https://imgur.com/a/oSt0itU

Brave
https://imgur.com/a/NrGeYJV

The extra step we add in this case is troubling and I'm not sure necessary.

**Comment by jumde** [comment]
From discussion with @anthonypkeane and @jhreis - We should be able to intercept and suppress the second prompt to avoid DoS by `tel:` permission prompt. If we can do that, removing the first prompt makes sense.

**Comment by jhreis** [comment]
To add additional clarity, the `tel:` scheme, already requires confirmation before performing a call, so we can remove the Brave one and let iOS handle it.

**Comment by anthonypkeane** [comment]
Correct Joel, I am proposing that.
@jumde need to give the ok from security before proceeding

**Comment by jhreis** [comment]
👍 can confirm we can suppress the second prompt, so this is unblocked. There is another ticket for the actual DoS suppression: https://github.com/brave/brave-ios/issues/2460

**Comment by jumde** [comment]
Looks like this is unblocked, let me know if this is still blocked. cc: @jhreis @anthonypkeane
